examples/platform/silabs/BaseApplication.cpp

```diff
@@ -251,7 +251,10 @@ void BaseApplicationDelegate::OnCommissioningWindowClosed()
         SilabsLCD::Screen_e screen;
         slLCD.GetScreen(screen);
         VerifyOrReturn(screen == SilabsLCD::Screen_e::QRCodeScreen);
-        slLCD.SetScreen(SilabsLCD::Screen_e::DemoScreen);
```

Comment:
Should this line be deleted? How is this done now I don't see it

Reply:
Adding a variable to pass with the AppEvent, the SetScreen is done a part of the UpdateDisplay-> cycle screen

Reply:
check, whether we got any data if it is nullptr cycle screen